docs/completion.md

```diff
@@ -1,15 +1,20 @@
-# Completion
+# Shell completion
+
+To make calling `awscnfm` actions easier and quicker, tab completion for shell commands is supported.
+
+## ZSH
 
 If you are using Oh My ZSH you need to put the generated completion into the
 plugins folder.
 
-```
-mkdir -p ~/.oh-my-zsh/custom/plugins/awscnfm && awscnfm completion zsh > ~/.oh-my-zsh/custom/plugins/awscnfm/_awscnfm
+```nohighlight
+mkdir -p ~/.oh-my-zsh/custom/plugins/awscnfm
+awscnfm completion zsh > ~/.oh-my-zsh/custom/plugins/awscnfm/_awscnfm
```

Comment:
This just makes it more complex and tedious to just move on. I am a developer and I want a one liner. Now it is two lines and "a waste of my time". ❤️

Reply:
I think you can easily copy and paste two lines.

My intention was to improve readability. Since `mkdir -p` won't fail if the directory is in place already, there is also no need for the `&&` operator.